components/style/values/computed/mod.rs

```diff
@@ -494,6 +495,24 @@ impl SVGPaint {
 /// <length> | <percentage> | <number>
 pub type LengthOrPercentageOrNumber = Either<Number, LengthOrPercentage>;
 
+#[cfg(feature = "gecko")]
+impl ToAnimatedValue for LengthOrPercentageOrNumber {
```

Comment:
@mantaroh see #17053 
It noticed me that if we convert Number to LengthOrPecentage in ToAnimatedValue, it will result glitch animations. E.g. very small strok-dasharray values with a big scale value for transform.
So, I think we should figure out another way, anyway.  (I think Au needs more precision)